src/traces/icicle/attributes.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var hovertemplateAttrs = require('../../plots/template_attributes').hovertemplateAttrs;
+var texttemplateAttrs = require('../../plots/template_attributes').texttemplateAttrs;
+
+var colorScaleAttrs = require('../../components/colorscale/attributes');
+var domainAttrs = require('../../plots/domain').attributes;
+var pieAttrs = require('../pie/attributes');
+var sunburstAttrs = require('../sunburst/attributes');
+var treemapAttrs = require('../treemap/attributes');
+var constants = require('../treemap/constants');
+var extendFlat = require('../../lib/extend').extendFlat;
+
+module.exports = {
+    labels: sunburstAttrs.labels,
+    parents: sunburstAttrs.parents,
+
+    values: sunburstAttrs.values,
+    branchvalues: sunburstAttrs.branchvalues,
+    count: sunburstAttrs.count,
+
+    level: sunburstAttrs.level,
+    maxdepth: sunburstAttrs.maxdepth,
+
+    tiling: {
+        orientation: {
+            valType: 'enumerated',
+            values: ['v', 'h'],
+            dflt: 'h',
+            editType: 'plot',
+            description: [
+                'Sets the orientation of the icicle.',
+                'With *v* the icicle grows vertically.',
```

Comment:
this isn't really clear to me... "v" means "bottom up" and "h" means "left to right" ? It would be helpful to note here that the other two possibilities (top-down and right-to-left) can be achieved via `flip` if that is indeed how to do it.

Reply:
In fact I might phrase this as "the root node(s) are on the left" rather than talking about icicles "growing" :)

Reply:
Great point, and I agree with this. I'l add that note.